installer/PowerToysSetup/Product.wxs

```diff
@@ -471,9 +471,7 @@
 
     <DirectoryRef Id="ImageResizerInstallFolder" FileSource="$(var.BinX64Dir)modules\$(var.ImageResizerProjectName)">
       <Component Id="Module_ImageResizer" Guid="96E63289-759C-4A73-A56B-EE7429932F72" Win64="yes">
-        <File Source="$(var.BinX64Dir)modules\$(var.ImageResizerProjectName)\ImageResizer.exe">
-          <netfx:NativeImage Id="ImageResizer.exe" Platform="all" Priority="0" />
-        </File>
+        <File Source="$(var.BinX64Dir)modules\$(var.ImageResizerProjectName)\ImageResizer.exe" />
```

Comment:
After this is removed, do we still need the following?
https://github.com/microsoft/PowerToys/blob/3f728efbf1ff51adb511b990f0b0932a2e3d3fc5/installer/PowerToysSetup/Product.wxs#L4
I don't have any idea, just asking.

Reply:
Not if we're not using that XML namespace anymore, and looks like we don't.

Reply:
OK, so let's remove it.